dailizabot/text_patterns.py

```diff
@@ -13,4 +13,20 @@
     "Würde {0} dir denn wirklich helfen?",
     "Bist du sicher, dass du {0} brauchst?"]],
 
+    [r"Du bist (.*)", 
+     ["Wieso denkst du das {0}?",
+      "Kann ich etas dagegen tun {0}?", 
+      "Dankeschön, dass ich {0} bin!"]]
```

Comment:
```suggestion
      "Dankeschön, dass ich {0} bin!"]],
```
The other patterns doesn't work, if there isn't a comma (it's a list)!